prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+   var fullName = 'Alan Kim';
+   console.log('fullName:', fullName);
+   var numberofPets = '1';
+   console.log('numberofPets:', numberofPets);
+   var likesVegetables = 'true';
+   console.log('likesVegetables:', likesVegetables);
+   var nothing = 'null';
```

Comment:
For numbers, booleans, and null, we'll want to leave off the quotes so that the javascript engine doesn't interpret them as strings

Reply:
![edited prep-javascript-variables](https://user-images.githubusercontent.com/97194926/204067097-d22a7e04-5dd3-4066-81fa-bc44f6768b65.png)